File: College-Sports-Domain/Models/Video.cs
using System.Collections.Generic; 
using System; 
namespace College_Sports_WebApp.Database.Models{ 

    public class Video
    {
        public string source { get; set; }
        public int id { get; set; }
        public string dataSourceIdentifier { get; set; }
        public string headline { get; set; }
        public string caption { get; set; }
        public string description { get; set; }
        public bool premium { get; set; }
        public Ad ad { get; set; }
        public Tracking tracking { get; set; }
        public string cerebroId { get; set; }
        public string contributingPartner { get; set; }
        public string contributingSystem { get; set; }
        public DateTime lastModified { get; set; }
        public DateTime originalPublishDate { get; set; }
        public TimeRestrictions timeRestrictions { get; set; }
        public DeviceRestrictions deviceRestrictions { get; set; }
        public bool syndicatable { get; set; }
        public int duration { get; set; }
        public List<Category> categories { get; set; }
        public List<string> keywords { get; set; }
        public PosterImages posterImages { get; set; }
        public List<Image> images { get; set; }
        public string thumbnail { get; set; }
        public string title { get; set; }
    }

}

File: College-Sports-WebApp/ClientApp/src/api/models/video.ts
/* tslint:disable */
/* eslint-disable */
import { Ad } from '../models/ad';
import { DeviceRestrictions } from '../models/device-restrictions';
import { Image } from '../models/image';
import { PosterImages } from '../models/poster-images';
import { TimeRestrictions } from '../models/time-restrictions';
import { Tracking } from '../models/tracking';
export interface Video {
  ad?: Ad;
  caption?: string | null;
  cerebroId?: string | null;
  contributingPartner?: string | null;
  contributingSystem?: string | null;
  dataSourceIdentifier?: string | null;
  description?: string | null;
  deviceRestrictions?: DeviceRestrictions;
  duration?: number;
  headline?: string | null;
  id?: number;
  images?: Array<Image> | null;
  keywords?: Array<string> | null;
  lastModified?: string;
  originalPublishDate?: string;
  posterImages?: PosterImages;
  premium?: boolean;
  source?: string | null;
  syndicatable?: boolean;
  thumbnail?: string | null;
  timeRestrictions?: TimeRestrictions;
  title?: string | null;
  tracking?: Tracking;
}
